bagitobjecttransfer/recordtransfer/tests/unit/test_views.py

```diff
@@ -287,11 +272,13 @@ def test_list_uploaded_files_empty_session(self):
 
     def test_list_uploaded_files_with_files(self):
         """Session has one file."""
-        uploaded_file = UploadedFile(
+        uploaded_file = TempUploadedFile(
             session=self.session,
             file_upload=SimpleUploadedFile("testfile.txt", self.one_kib),
             name="testfile.txt",
         )
+        self.session.status = UploadSession.SessionStatus.UPLOADING
+        self.session.save()
```

Comment:
What do you think about doing this instead?

```suggestion
        self.session.add_temp_file(SimpleUploadedFile("testfile.txt", self.one_kib))
```